### PR TITLE
quic: add comment codes and other cleanups

### PR DESCRIPTION
**Pull request by jasnell**
Adding code comments

### CODE REVIEW

File: src/node_quic_session.cc [hunk 1]
```diff
@@ -2135,6 +2186,14 @@ bool QuicSession::SendConnectionClose() {
   }
 }
 
+// When a server advertises a preferred address in it's initial
```
Comment (addaleax) [inline src/node_quic_session.cc]:
```suggestion
// When a server advertises a preferred address in its initial
```

File: src/node_quic_session.cc [hunk 2]
```diff
@@ -2199,6 +2261,7 @@ bool QuicSession::SendStreamData(QuicStream* stream) {
   return application_->SendStreamData(stream);
 }
 
+// Passes a serialized packet to the assocaited QuicSocket.
```
Comment (addaleax) [inline src/node_quic_session.cc]:
```suggestion
// Passes a serialized packet to the associated QuicSocket.
```

File: src/node_quic_session.cc [hunk 3]
```diff
@@ -3042,7 +3200,10 @@ int QuicSession::OnExtendMaxStreamData(
 
 // Called by ngtcp2 for both client and server connections
 // when ngtcp2 has determined that the TLS handshake has
-// been completed.
+// been completed. It is important to understand that this
+// is only an indication of the local peer's handshake state.
+// The remote peer might not yet have completed it's part
```
Comment (addaleax) [inline src/node_quic_session.cc]:
```suggestion
// The remote peer might not yet have completed its part
```

**Comment by jasnell** [comment]
Landed

**Pull request closed without merging**